bodhi/server/util.py

```diff
@@ -297,9 +298,23 @@ def hostname(context=None):
 
 
 def markup(context, text):
-    return markdown.markdown(text, safe_mode="replace",
-                             html_replacement_text="--RAW HTML NOT ALLOWED--",
-                             extensions=['markdown.extensions.fenced_code'])
+    """
+    Return HTML from a markdown string
+
+    Args:
+        context:
+        text (basestring): markdown text to be converted to HTML
+
+    Return:
+        basestring: HTML representation of the markdown text
+    """
+
+    # previously, we used the Safe Mode in python-markdown to strip all HTML
+    # tags. Safe Mode is deprecated, so we now use Bleach to sanitize all HTML
+    # tags before running it through the markdown parser
+    sanitized_text = bleach.clean(text, tags=[])
```

Comment:
Since bodhi itself is in EPEL, we shouldn't require a newer bleach than is available in EPEL. However, Bodhi is also in Rawhide. Rawhide currently has bleach 1.5, but I would assume it could get updated at any time.